config/metasrv.example.toml

```diff
@@ -29,6 +29,11 @@ store_key_prefix = ""
 max_retry_times = 12
 # Initial retry delay of procedures, increases exponentially
 retry_delay = "500ms"
+# Auto split large value
+# The etcd the maximum size of any request is 1.5 MiB
```

Comment:
Add some comment like: "GreptimeDB procedure uses etcd as the default metadata storage backend"